### DISCUSSION TITLE
Control your music in Waze

### DISCUSSION THREAD

**Discussion by sklado80** [TortugaPower/BookPlayer#690] [Ideas]
## Feature request

Please, add BookPlayer app to the list of supported Waze's audio players.
More info https://support.google.com/waze/answer/9171270
Thank you.

**Comment by pichfl** [comment]
This would require us [Become a Waze partner](https://developers.google.com/waze/intro-audio) and integrate the proprietary Waze toolkit into BookPlayer. 

I do understand that this can be a useful feature, but from an ideological standpoint adding proprietary third party code to our open source project feels iffy. It's also unclear if this would affect our privacy policy or allow Google to track our users in some way.
